README.md

```diff
@@ -67,9 +65,9 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- This design is similar to the one created by Cindy Shin .
+- Microverse
+
 
 ## 📝 License
```

Comment:
[OPTIONAL] I think the MIT license is broken. Please, fix it before merging the pull request.